src/ext.rs

```diff
@@ -130,9 +133,14 @@ pub fn balance(address: &Address) -> U256 {
 pub fn create(endowment: U256, code: &[u8]) -> Result<Address, Error> {
 	let mut endowment_arr = [0u8; 32];
 	endowment.to_big_endian(&mut endowment_arr);
-	let mut result = Address::new();
+	let mut result = Address::from(H160::zero());
```

Comment:
Shouldn't `Address::zero()` work here (but I do wish it was just `Address::new()`)?

Reply:
see above

Reply:
The `Hash::new` method was deprecated in version `0.2` and finally removed in `0.3.0-beta` of `fixed-hash`. Users should use `Hash::zero` instead.

Reply:
done